configuration/outputs.md

```diff
@@ -259,6 +259,66 @@ If enabled, the Elastic Search integration will stream analytics and insights to
         #password: "elastic search password"
 ```
 
+## Azure
+
+If enabled, the Azure integration will stream analytics and insights to an Azure endpoint.
+
+| Key | Description | Required |
+| :--- | :--- | :--- |
+| name | User defined name of this specific destination, used for mapping this destination to a workflow | Yes |
+| type | Streaming destination type \(azure\) | Yes |
+| endpoint | Azure AppInsight endpoint. | Yes |
+| api_key | Azure AppInsight API key. | Yes |
+| features | Features defines which data types stream to backend, it can be "log", "metric", "edac", "cluster", "topk" or "all". If you don't provide any value then it is all. | No |
```

Comment:
should we list out which features are supported by each pusher here. AFAIK not all pusher can push same features.

Reply:
+1. also a dedicated section for what each feature mean and which processors generate which one etc. would be super useful